<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeInterface</class>
 <widget class="QWidget" name="HomeInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QFrame" name="HomeFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>576</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QGroupBox" name="SummonerInfoGroupBox">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>5</y>
      <width>302</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="SummonerIconLabel">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="SummonerNameLabel">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>5</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="SummonerRegionLabel">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>35</y>
       <width>115</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="SummonerRankedLabel">
     <property name="geometry">
      <rect>
       <x>183</x>
       <y>35</y>
       <width>115</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>无段位</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="SummonerExpeProgressBar">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>76</y>
       <width>180</width>
       <height>8</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="SummonerExpeTextLabel">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>68</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>经验值</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="SummonerLevelLabel">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>68</y>
       <width>64</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="FriendListWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>38</y>
      <width>220</width>
      <height>534</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScrollMargin">
     <number>1</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectItems</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="uniformItemSizes">
     <bool>true</bool>
    </property>
    <property name="batchSize">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="FriendListTextLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>好友列表：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="RankDataTableWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>127</y>
      <width>393</width>
      <height>445</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QTableWidget" name="RankFightDataTableWidget">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>127</y>
      <width>393</width>
      <height>445</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="RankDataListTextLabel">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>99</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>排位榜：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="RankFightDataTextLabel">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>99</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>乱斗榜：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NoteTextLabel">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>5</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>注意事项：
1.榜单数据来自101.qq.com
2.观战只能观看当前联盟大区
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="FindSummonerNameLineEdit">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>67</y>
      <width>180</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="AddFriendPushButton">
    <property name="geometry">
     <rect>
      <x>671</x>
      <y>67</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="text">
     <string>添加好友</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FindSummonerTagLineEdit">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>67</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="SpectatePushButton">
    <property name="geometry">
     <rect>
      <x>586</x>
      <y>67</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="text">
     <string>观看对局</string>
    </property>
   </widget>
   <widget class="QLabel" name="FindSummonerTagSymbolLabel">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>67</y>
      <width>15</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>#</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../ResourceFiles/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
